src/compositions/templates/case-study/MetricGrid/index.jsx

```diff
@@ -4,19 +4,15 @@ import PropTypes from 'prop-types';
 
 import { Grid } from './styled';
 
-const MetricGrid = ({ metrics }) => (
+const MetricGrid = ({ content }) => (
   <Grid>
-    {metrics.map((metric, i) => (
-      <Metric {...metric} max={metrics.length} i={i} />
-    ))}
+    {content &&
```

Comment:
I think using `defaultProps` like it was before, instead of having `{content && ...}` is cleaner and more declarative.